.travis.yml

```diff
@@ -34,7 +34,7 @@ install:
 
 script:
   - "pyflakes src"
-  - "coverage run --rcfile=${PWD}/.coveragerc $(type -p trial) txkube"
+  - "coverage run --rcfile=${PWD}/.coveragerc python -Wdefault::DeprecationWarning -m twisted.trial txkube"
```

Comment:
What version of Twisted began providing `python -m twisted.trial`?  Should we add a constraint to setup.py?

Reply:
This was introduced in https://github.com/twisted/twisted/commit/2ee449097bc8132c1645d300a273356d0d0f97c8
which went in Twisted 16.4.0